test/TestHelper.cpp

```diff
@@ -280,6 +280,11 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 	}
 }
 
+void ImportTest::importTransaction(json_spirit::mObject& o_tr)
```

Comment:
this function does not return any value. 
it is a method of class ImportTest that imports transaction from json object